segmentstore/server/host/src/main/java/io/pravega/segmentstore/server/host/handler/PravegaRequestProcessor.java

```diff
@@ -502,6 +503,10 @@ public void createSegment(CreateSegment createStreamSegment) {
     }
 
     private CompletableFuture<Void> createIndexSegment(final String segmentName) {
+        if (!isUserStreamSegment(segmentName)) {
+            log.info("No need to create index segment for segment {}.", segmentName);
```

Comment:
Should we have this log in Info, better to enable debug level.

Reply:
Create segment call is not very frequent, so i think we  can keep it at info level.